test/ownership/HasNoEther.test.js

```diff
@@ -45,7 +45,7 @@ contract('HasNoEther', function (accounts) {
     const ownerFinalBalance = await ethGetBalance(accounts[0]);
     const finalBalance = await ethGetBalance(hasNoEther.address);
     assert.equal(finalBalance, 0);
-    assert.isAbove(ownerFinalBalance, ownerStartBalance);
+    assert.isTrue(ownerFinalBalance.greaterThan(ownerStartBalance));
```

Comment:
There is a separate issue (#1091) for this, try to only commit relevant changes :)